packages/e2e-cypress/src/install.js

```diff
@@ -105,6 +105,8 @@ module.exports = async function (api) {
     devServerPort,
     shouldAddCodeCoverage,
     shouldSupportTypeScriptAndVite: shouldSupportTypeScript && api.hasVite,
+    // See https://github.com/quasarframework/quasar-testing/issues/379
+    requiresPublicPath: api.hasVite && api.hasPackage('@quasar/app-vite', '^2.0.0-0')
```

Comment:
```suggestion
    requiresPublicPath: api.hasVite && api.hasPackage('@quasar/app-vite', '^2.0.0')
```

Reply:
The aim here was to support  `@quasar/app-vite` pre-release versions. Maybe `>=2.0.0-0 <3.0.0` should be better OR don't we need to? In which case we need to upgrade the `test-vite-2` app. It's currently using a pre-release version. I plan to update it in a different PR.

Reply:
Sounds good.